private/protocol/xml/xmlutil/unmarshal.go

```diff
@@ -52,14 +54,26 @@ func parse(r reflect.Value, node *XMLNode, tag reflect.StructTag) error {
 	if t == "" {
 		switch rtype.Kind() {
 		case reflect.Struct:
-			t = "structure"
+			// also it can't be a time object
+			_, tok := r.Interface().(*time.Time)
```

Comment:
May be cleaner to do something like this
```go
if _, ok := reflect.Indirect(v).Interface().(time.Time); ok {
}
```

Reply:
Wouldn't be able to use `Indirect` because that would produce a nil value for fields, which `Interface` will panic on.